doc/over_sampling.rst

```diff
@@ -211,6 +211,44 @@ Therefore, it can be seen that the samples generated in the first and last
 columns are belonging to the same categories originally presented without any
 other extra interpolation.
 
+However, :class:`SMOTENC` is working with data composed of categorical data
```

Comment:
However, :class:`SMOTENC` is working with datasets composed of continuous and categorical features.